ketcher-autotests/tests/Structure-Creating-&-Editing/Actions-With-Structures/Rotation/rotation.spec.ts

```diff
@@ -0,0 +1,38 @@
+import { expect, test } from '@playwright/test';
+import {
+  getEditorScreenshot,
+  drawBenzeneRing,
+  selectSingleBondTool,
+} from '@utils';
+import { getRotationHandleCoordinates } from '@utils/clicks/selectButtonByTitle';
+
+test.describe('Rotation', () => {
+  const MOUSE_COORDS = {
+    x: 20,
+    y: 160,
+  };
+
+  test.beforeEach(async ({ page }) => {
+    await page.goto('');
+    await drawBenzeneRing(page);
+    await selectSingleBondTool(page);
+  });
+
+  test('Cancel rotation on right click', async ({ page }) => {
+    const snapshotBeforeRotation = await getEditorScreenshot(page);
```

Comment:
I don't understand this test. First of all, why do you need to choose `selectSingleBondTool`?
Secondly, you do literally nothing with the structure. 
1) You get coordinates of rotation handle
2) Then you click on it – no action is performed on the structure
3) Then you move mouse to some coordinates (may be worth renaming them to something like `COORDINATES_TO_PERFORM_ROTATION`)
4) And then make a right click.

This way, test is supposed to be:
1) You get coordinates of rotation handle.
2) `mousedown` on rotation handle, not click
3) Move mouse to some coordinates (you need to drag handle, not simply click on it)
4) Press right click